test/e2e/storage/drivers/csi.go

```diff
@@ -354,6 +355,7 @@ func InitGcePDCSIDriver() testsuites.TestDriver {
 				testsuites.CapExec:        true,
 				testsuites.CapMultiPODs:   true,
 			},
+			RequiredAccessModes: []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce},
```

Comment:
It would be safe to delete RequiredAccessModes for GcePDCSIDriver, because RWO is default now.
Instead, we should add ROX and RWX to drivers that support them, properly. 
(I hope that https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes is still up-to-date and correct.)

Although, this change could be covered by another PR that actually tests ROX and RWX features. (As PR says  just "add a field", not "set the field" for all drivers.)

Reply:
> It would be safe to delete RequiredAccessModes for GcePDCSIDriver, because RWO is default now.
Instead, we should add ROX and RWX to drivers that support them, properly.
(I hope that https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes is still up-to-date and correct.)

It would be safe, but technically more correct to set this explicitly. Defaulting can change and is best not to be relied on :)

>Although, this change could be covered by another PR that actually tests ROX and RWX features. (As PR says just "add a field", not "set the field" for all drivers.)

Agreed, setting this field can be done as a follow up.